doc/mcuboot/readme-ncs.rst

```diff
@@ -26,7 +26,10 @@ See the `MCUboot output build files`_ page for a list of all these files.
 
 .. note::
```

Comment:
This note has become absolutely massive and I would argue it should be converted to a section titled 'Using MCUboot in the direct-xip mode' or something similar.